src/service.js

```diff
@@ -0,0 +1,316 @@
+if(!global._babelPolyfill) { require('babel-polyfill'); }
+
+import Proto from 'uberproto';
+import mongoose from 'mongoose';
+import filter from 'feathers-query-filters';
+import errors from 'feathers-errors';
+import errorHandler from './error-handler';
+
+// Use native promises
+mongoose.Promise = global.Promise;
+
+const Schema = mongoose.Schema;
```

Comment:
Hi, I'm new here, but using feathers-mongoose on a new project. My 2 cents, I think having the "user" pass in their own connection is a better idea, and a better practice in general. I think the default of having a new connection made automatically will have little to no use in the wild, IMHO. An example of that would probably be super helpful as well in the first example. I would be happy to add it if it doesn't make it into this pull request.